SPAN_KIND macros undefined

Should those ?SPAN_KIND macros come from opentelemetry or be local to the Zipkin app? This seems to come from the switch to protobuf.

```
===> Compiling opentelemetry_api
===> Compiling opentelemetry
===> Compiling opentelemetry_zipkin
===> Compiling _build/default/lib/opentelemetry_zipkin/src/opentelemetry_zipkin.erl failed
_build/default/lib/opentelemetry_zipkin/src/opentelemetry_zipkin.erl:131: undefined macro 'SPAN_KIND_UNSPECIFIED'

_build/default/lib/opentelemetry_zipkin/src/opentelemetry_zipkin.erl:74: function to_kind/1 undefined

make: *** [Makefile:26: rel] Error 1
```
I've adapted to the "otel" module names here: https://github.com/ioolkos/opentelemetry_zipkin/commit/cfbbee5f2adfcc1082aa3999bc634dded840fff1, but that's unrelated.
Thanks so much!

--- a/rebar.lock
+++ b/rebar.lock
@@ -1,8 +1,11 @@
-{"1.1.0",
-[{<<"opentelemetry">>,{pkg,<<"opentelemetry">>,<<"0.4.0">>},0},
- {<<"opentelemetry_api">>,{pkg,<<"opentelemetry_api">>,<<"0.3.1">>},0}]}.
+{"1.2.0",
+[{<<"opentelemetry">>,{pkg,<<"opentelemetry">>,<<"0.4.1">>},0},
+ {<<"opentelemetry_api">>,{pkg,<<"opentelemetry_api">>,<<"0.4.1">>},0}]}.
 [
 {pkg_hash,[
- {<<"opentelemetry">>, <<"293729F014009B03A1A2C47E6367DB6F280B41412FAA5639F06DCCE9733D18A6">>},
- {<<"opentelemetry_api">>, <<"AA042F9FF0B774C3E9827E215FCF972D5CFCCD9A79ED55194B58C329A945B486">>}]}
+ {<<"opentelemetry">>, <<"519F0E2DAA89D1C4A0FB37310DB9828EE89181E6CFF349F6B1FDBDA2FC4441E5">>},
+ {<<"opentelemetry_api">>, <<"AFCE47063E099407DAA105BF25E2AAE75822F3C25B61A213F9328FF40AE6C448">>}]},
+{pkg_hash_ext,[
+ {<<"opentelemetry">>, <<"AFBA124EBE368FD530DB30347EC3BF41F489A3D9B245BFD8397D5753B83E3AAE">>},
+ {<<"opentelemetry_api">>, <<"7B057CA9BD5F2220D02FE2337630979DF27C8D0FEFC6EBA371FB3CB202EB618E">>}]}
 ].

--- a/src/opentelemetry_zipkin.erl
+++ b/src/opentelemetry_zipkin.erl
@@ -122,8 +122,6 @@ to_parent_id(ParentId) ->
 
 to_kind(undefined) ->
     'SPAN_KIND_UNSPECIFIED';
-to_kind(?SPAN_KIND_UNSPECIFIED) ->
-    'SPAN_KIND_UNSPECIFIED';
 to_kind(?SPAN_KIND_INTERNAL) ->
     'SPAN_KIND_UNSPECIFIED';
 to_kind(?SPAN_KIND_PRODUCER) ->

--- a/test/opentelemetry_zipkin_SUITE.erl
+++ b/test/opentelemetry_zipkin_SUITE.erl
@@ -43,6 +43,7 @@ verify_export(_Config) ->
                         #event{system_time_nano=erlang:system_time(nanosecond),
                                name = <<"event-2">>,
                                attributes = [{<<"attr-3">>, <<"value-3">>}]}],
+              status=opentelemetry:status(?SPAN_KIND_INTERNAL, <<"some message about status">>),
               attributes = [{<<"attr-2">>, <<"value-2">>}]},
     true = ets:insert(Tid, ParentSpan),
 

